Unable to initialize a new project using the ignite CLI

### Describe the bug

I am trying to initialize a new project using the CLI, having the following error with it. 

The following error occurred:

```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: quick-test-ignite-cli@0.0.1
npm error Found: eslint-plugin-n@17.11.1
npm error node_modules/eslint-plugin-n
npm error   dev eslint-plugin-n@"^17.10.2" from the root project
npm error
npm error Could not resolve dependency:
npm error peer eslint-plugin-n@"^15.0.0 || ^16.0.0 " from eslint-config-standard@17.1.0
npm error node_modules/eslint-config-standard
npm error   dev eslint-config-standard@"^17.1.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\hp\AppData\Local\npm-cache\_logs\2024-10-29T16_26_12_983Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\hp\AppData\Local\npm-cache\_logs\2024-10-29T16_26_12_983Z-debug-0.log
```

Using the --legacy-peer-deps & --force also did not work as well. 


### Ignite version

Ignite 10.0.1

### Additional info

### System
  platform           win32
  arch               x64
  cpu                8 cores              Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
  directory          QuickTestIgniteCli   C:\Users\hp\Desktop\Development\QuickTestIgniteCli

### JavaScript (and globally-installed packages)
  node                          20.12.0             C:\Program Files\nodejs\node.EXE
  npm                           10.8.2              C:\Program Files\nodejs\npm.CMD
    @superforms/superforms-rn   0.0.0-development
    jetifier                    2.0.0
    npm                         10.8.2
  yarn                          -                   not installed
  pnpm                          -                   not installed
  bun                           -                   not installed
  expo                          -                   not installed

### Ignite
  ignite-cli         10.0.1       C:\Users\hp\AppData\Local\npm-cache\_npx\bec2839915ba8fc3\node_modules\.bin\ignite.CMD

  ignite src         build        C:\Users\hp\AppData\Local\npm-cache\_npx\bec2839915ba8fc3\node_modules\ignite-cli\build

### Android
  java               17.0.10      C:\Program Files\Microsoft\jdk-17.0.10.7-hotspot\bin\java.EXE
  android home       -            C:\Users\hp\AppData\Local\Android\Sdk

### Tools
  git                git version 2.44.0.windows.1   C:\Program Files\Git\cmd\git.EXE

@frankcalise Tried the latest:10.0.3 it failed on the clean up. 

```
    █ Creating QuickTestIgniteCliv2 using Ignite 10.0.3
    █ Powered by  ∞ Infinite Red  (https://infinite.red)
    █ Package Manager: npm
    █ Bundle identifier: com.quicktestignitecliv2
    █ Path: C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2
    ────────────────────────────────────────────────────

    🖨  3D-printing a new React Native app
    🎨 Getting those last few details perfect
    🧶 Installing npm dependencies (wow these are heavy)
    ⚙️ Configuring app.json
    🛠️ Generating native template via Expo Prebuild
    🛠️ Removing fancy demo code
    🌳 Removing MobX-State-Tree code
    ❌ Cleaning up

The following error occurred:


> quick-test-ignite-cliv-2@0.0.1 format
> eslint . --fix


Error while parsing C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\WelcomeScreen.tsx
Line 22, column 2: '{' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

Error while parsing C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\WelcomeScreen.tsx
Line 22, column 2: '{' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

Error while parsing C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\WelcomeScreen.tsx
Line 22, column 2: '{' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

Error while parsing C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\WelcomeScreen.tsx
Line 22, column 2: '{' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

Error while parsing C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\WelcomeScreen.tsx
Line 22, column 2: '{' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

Error while parsing C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\app.tsx
Line 109, column 2: Declaration or statement expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored

C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\App.tsx
  3:17  error  Parse errors in imported module '@/app': Declaration or statement expected. (109:2)  import/namespace

C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\app.tsx
  109:2  error  Parsing error: Declaration or statement expected

C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\navigators\AppNavigator.tsx
  63:55  error  'WelcomeScreen' not found in imported namespace 'Screens'  import/namespace

C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\WelcomeScreen.tsx
  22:2  error  Parsing error: '{' expected

C:\Users\hp\Desktop\Development\QuickTestIgniteCliv2\app\screens\index.ts
  1:15  error  Parse errors in imported module './WelcomeScreen': '{' expected. (22:2)  import/export

✖ 5 problems (5 errors, 0 warnings)

```

Trying 10.0.2, I will get back to you. Also, I tried it on my work laptop (mac) as well. It had issues as well. I will try to share the log on it as well.

--- a/boilerplate/app/app.tsx
+++ b/boilerplate/app/app.tsx
@@ -101,9 +101,8 @@ function App(props: AppProps) {
   // In iOS: application:didFinishLaunchingWithOptions:
   // In Android: https://stackoverflow.com/a/45838109/204044
   // You can replace with your own loading component if you wish.
-  // @mst replace-next-line if (!isNavigationStateRestored || (!areFontsLoaded && !fontLoadError)) {
   if (
-    !rehydrated ||
+    !rehydrated || // @mst remove-current-line
     !isNavigationStateRestored ||
     !isI18nInitialized ||
     (!areFontsLoaded && !fontLoadError)

--- a/boilerplate/app/screens/WelcomeScreen.tsx
+++ b/boilerplate/app/screens/WelcomeScreen.tsx
@@ -21,7 +21,7 @@ interface WelcomeScreenProps extends AppStackScreenProps<"Welcome"> {}
 
 // @mst replace-next-line export const WelcomeScreen: FC<WelcomeScreenProps> = (
 export const WelcomeScreen: FC<WelcomeScreenProps> = observer(
-  function WelcomeScreen(
+  function WelcomeScreen( // @mst remove-current-line
     _props, // @demo remove-current-line
     // @mst replace-next-line ) => {
   ) {
@@ -81,7 +81,7 @@ export const WelcomeScreen: FC<WelcomeScreenProps> = observer(
     )
     // @mst replace-next-line }
   },
-)
+) // @mst remove-current-line
 
 const $topContainer: ThemedStyle<ViewStyle> = ({ spacing }) => ({
   flexShrink: 1,

